ReqIFSharp/AttributeValue/AttributeValueBoolean.cs

```diff
@@ -64,7 +64,7 @@ internal AttributeValueBoolean(SpecElementWithAttributes specElAt)
         /// <summary>
         /// Gets or sets the attribute value.
         /// </summary>
-        public bool TheValue { get; set; }
+        public bool? TheValue { get; set; }
```

Comment:
I don't think we should make this nullable. According to the spec this value should be set. Like this we give API developers the idea this is optional, and it is not. Please revert

Reply:
Agree with you here